vm/src/compile.rs

```diff
@@ -24,6 +24,7 @@ struct Compiler {
     current_qualified_path: Option<String>,
     in_loop: bool,
     in_function_def: bool,
+    in_exc_handler: bool,
```

Comment:
This looks like a thing that should be handled at runtime, rather than at compile time. As of now, while this code does handle
```python
try:
	1/0
except:
	raise
```
, it doesn't handle anything more complex, like
```python
def f():
    raise

try:
    1/0
except:
    f()
```

Reply:
I agree. CPython has a stack of exceptions so we can do something similar.